server/routes/middleware/outcomesMiddleware.js

```diff
@@ -14,12 +14,13 @@ const outcomesMiddleware = state => async (req, res, next) => {
   }
   const hearingDateSort = sortMapping[sorts && sorts.map(item => item.value).pop()]
 
+  // TODO: get these counts from the API
```

Comment:
But you wouldn't make the API call here? Probably you could?

Reply:
The idea was that I would do it here in the middleware, then each controller can update it's own count from the applied filters etc